src/controls/LayerControl.js

```diff
@@ -44,14 +32,12 @@ const baseLayers = [
     image: orthoThumbnail,
     layer: new TileLayer({
       visible: false,
-      source: new XYZ(
-        Object.assign({
-          attributions: bmapAttribution,
-          crossOrigin: 'anonymous',
-          tileGrid: bmapTilegrid,
-          url: 'https://neu{1-4}.mapserver.at/mapproxy/wmts/bmaporthofoto30cm/normal/google3857/{z}/{y}/{x}.jpeg',
-        }),
-      ),
+      source: new ImageTile({
+        maxZoom: 19,
+        url: 'https://mapsneu.wien.gv.at/basemap/bmaporthofoto30cm/normal/google3857/{z}/{y}/{x}.jpg',
+        attributions:
+          '<a href="http://www.basemap.at">basemap.at</a> &copy; <a href="http://creativecommons.org/licenses/by/3.0/at/">CC BY 3.0 AT</a>',
```

Comment:
```suggestion
          'Grundkarte: <a href="https://basemap.at">basemap.at</a>',
```
This is the correct attribution for basemap.at according to https://basemap.at/#lizenz.